docs/SOPs/wrangling_best_practices.md

```diff
@@ -0,0 +1,369 @@
+* Disease here is almost more important than the donor level disease. Make sure to include information.
+    * Donor could have HepC, but the specimen they donate is still `normal`
+* Some specimens are better listed as systems, instead of organs
+    * Some datasets in the dcp label organ = bone marrow and others use organ=hematopoietic system/organ_part = bone marrow
+
+### Cell Suspension
+
+* For SS2 datasets, set the `estimated cell count` to 1 for each cell suspension
+    * Well and plate number are nice to have for QC purposes
+* The input biomaterial can be any of the following depending on the experiment:
+    * Specimen (single cell/nuclei OR bulk)
+    * Cell line
+    * Organoid
+    * Cell suspension (WIP; please see this [ticket](https://app.zenhub.com/workspaces/operations-5fa2d8f2df78bb000f7fb2b5/issues/ebi-ait/hca-ebi-wrangler-central/927))
+    
+### Organoids
+
+* Some are made via differentiation protocols, others are made by growing multipotent cells together which signal differentiation without an applied protocol
+    * Example datasets
+        * Differentiation included: [HumanCerebralOrganoidsFetalNeocortex](https://contribute.data.humancellatlas.org/projects/detail?uuid=d2111fac-3fc4-4f42-9b6d-32cd6a828267)
+* Input biomaterial can be:
+    * Specimen
+    * Cell line
+    
+### Cell line
+
+* Cell lines should always have information about the donor in the `Donor organism` tab.
+  * Example dataset: [pyleSkeletalMuscle](https://contribute.data.humancellatlas.org/projects/detail?uuid=4037007b-0eff-4e6d-b7bd-8dd8eec80143)
+* A cell line can make another cell line (also the input)
+    * Example dataset: [iPSCderivedTenocyte](https://contribute.data.humancellatlas.org/projects/detail?uuid=78d7805b-fdc8-472b-8058-d92cf886f7a4)
+* If the cell line was purchased, please use the name listed here: [Cellosaurus](https://web.expasy.org/cellosaurus). This helps with consistency within the database.
+* Organ model - for embryonic and pluripotent stem cells this is not ideal, but you can list `embryo` or `whole body` 
+* If it’s an embryonic cell line, the donor would be the female that donates the embryonic tissue.
+
+### Imaging
+
+* Visium Datasets are modelled with the following subgraphs:
+    * Donor_organism → specimen_from_organism → imaged_specimen → …
+    * Imaged_specimen → image_file
+    * Imaged_specimen → sequence_file
+* It’s helpful to use the Visium Spatial Gene Expression ontology term as the library preparation protocol to generate the Visium fastq files
+* FFPE vs Fresh-Frozen information can be stored in the `preservation method` term
+* Add the `permeabilization time` to the imaging_preparation_protocol
+
+## Files
+
+### All files
+
+* Always fill up the `file source`. This helps the downstream components to identify the CGMs.
+  * <span style="color:red">Do not use DCP/2 Ingest</span>. That term is reserved for the spreadsheet generated by ingest.
```

Comment:
Is this relevant for sequence files too?

Reply:
yep! it should be filled for all files, really, even though we don't usually do it